packages/uploadthing/src/internal/shared-schemas.ts

```diff
@@ -78,6 +78,7 @@ export class UploadedFileData extends FileUploadDataWithCustomId.extend<Uploaded
   key: S.String,
   url: S.String,
   appUrl: S.String,
+  hash: S.String,
```

Comment:
Is hash a good name here? fileHash? contentHash?

Reply:
I think fileHash might be better, more descriptive, good call